CHANGELOG.rst

```diff
@@ -0,0 +1,43 @@
+=========
+Changelog
+=========
+
+######
+Future
+######
+
+
+****************
+Breaking Changes
+****************
+
+******************************
+Major Feature and Improvements
+******************************
+
+  * Include `mgclient` to decouple pymgclient from the installed version of
+    `mgclient`, thus make the building and usage easier.
+
+*********
+Bug Fixes
+*********
+
+  * Fix various memory leaks.
+  * Fix transaction handling when an error happens in explicit transactional
+    mode. The running transaction is reset and a new one is going to be started
```

Comment:
```suggestion
    mode. The running transaction is reset and a new one is started
```

Reply:
I didn't applied the suggestion as is because of formatting. But the requested changes are done.